### ISSUE TITLE
Enable View Transition API

### ISSUE THREAD

**Issue by AllanOricil** [salesforce/lwc#3574]
https://developer.chrome.com/docs/web-platform/view-transitions/

Other major js frameworks have enabled it. Maybe this would be useful for lwc and could improve UX in the platform.

**Comment by nolanlawson** [comment]
Is there anything required for LWC users to take advantage of this API? I get the impression that Vue, React, etc. just added demos, not anything to the framework itself.

Unless you think [LWR](https://developer.salesforce.com/docs/platform/lwr/overview) should support this somehow?

**Comment by AllanOricil** [comment]
Yes. LWR could support it when changing routes (navigation). But I also think that there could exist support for components inside the same page when moving them around in the DOM.

**Comment by AllanOricil** [comment]
This LWR feature looks like a nuxt app. It would be nice if the routing could be automatically generated based on lwc components, like nuxt does.

**Comment by AllanOricil** [comment]
Nah.. it is not even close to nuxt. Wtf is that template language for defining layouts 😅
